pandas/tests/groupby/test_groupby.py

```diff
@@ -1882,3 +1882,69 @@ def test_groupby_axis_1(group_name):
     results = df.groupby(group_name, axis=1).sum()
     expected = df.T.groupby(group_name).sum().T
     assert_frame_equal(results, expected)
+
+
+@pytest.mark.parametrize("tz", [None, "Asia/Tokyo"])
```

Comment:
We have a `tz_naive_fixture` you can use in the function signature that will parameterize over more timezones.

```
def test_shift_bfill_ffill_tz(tz_naive_fixture, op, expected):
    tz = tz_naive_fixture
    ...
```